core/java/com/google/instrumentation/stats/View.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.instrumentation.stats;
+
+import com.google.instrumentation.common.Timestamp;
+
+import java.util.List;
+
+/**
+ * Container for the {@link Aggregation}s for a particular {@link ViewDescriptor}.
```

Comment:
You might want to get some more popcorn :)

More seriously, I'll try to come up with a more descriptive description.